satellite/console/consoleweb/server.go

```diff
@@ -0,0 +1,139 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package consoleweb
+
+import (
+	"context"
+	"encoding/json"
+	"net"
+	"net/http"
+	"path/filepath"
+
+	"github.com/graphql-go/graphql"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/auth"
+	"storj.io/storj/satellite/console"
+	"storj.io/storj/satellite/console/consoleweb/consoleql"
+)
+
+const (
+	authorization = "Authorization"
+	contentType   = "Content-Type"
+
+	authorizationBearer = "Bearer "
+
+	applicationJSON    = "application/json"
+	applicationGraphql = "application/graphql"
+)
+
+// Error is satellite console error type
+var Error = errs.Class("satellite console error")
+
+// Server represents console web server
+type Server struct {
+	logger *zap.Logger
+
+	config   Config
```

Comment:
Move `type Config struct` above `type Server struct`, it makes it easier to see what the config arguments are for this server.